tests/specs/logging/harvesting.e2e.js

```diff
@@ -81,7 +87,7 @@ describe('logging harvesting', () => {
         it(`should harvest expected logs - ${type} post load - logging mode: ${mode}`, async () => {
           await mockRumResponse(logLevel)
           const [[{ request: { body } }]] = await Promise.all([
-            logsCapture.waitForResult({ totalCount: 1 }),
+            logsCapture.waitForResult({ totalCount: 1, timeout: 15000 }),
```

Comment:
Just curious what was the reason for needing to add a timeout on the various logsCapture calls across some of these tests

Reply:
I wanted to speed up the test failures.  When I was working through the failures, it was taking a long time for the tests to timeout + fail.  15 secs seemed enough to determine if it was going to pass or not.